cli/src/test/resources/ide-projects/graalvm/repository/graalvm/graalvm/default/windows/bin/testFile.cmd

```diff
@@ -0,0 +1 @@
+echo graalvm windows %*
```

Comment:
this file is IMHO never used at all. Best practice is to avoid pointless clutter and only add test resources that are actually needed. You have a 2nd copy of it in the `default` edition. Also I wonder if we need the `community` edition at all. IMHO also the entire `repository/graalvm/community` folder is pointless and can be deleted.